### PR TITLE
Refactor:Card Comp

### PR DESCRIPTION
**Pull request by srishtigautam21**
Add: Filter section
Updated Link tag
File to check : component -> ProductFilter

### CODE REVIEW

File: src/pages/ProductPage/ProductPage.jsx [hunk 1]
```diff
@@ -1,14 +1,21 @@
 import { Card } from "../../component/Card/Card";
+import { ProductsFilter } from "../../component/ProductsFilter/ProductsFilter";
 import { useCard } from "../../component/Context/CardContext/CardContext";
```
Comment (yaswanthmyneni) [inline src/pages/ProductPage/ProductPage.jsx]: suggestion: add an index.js file in the component folder and export all components from index.js.

**Review by yaswanthmyneni — APPROVED** [review]
UI looks great, you can merge.

File: src/pages/ProductPage/ProductPage.jsx (hunk 1 above)
Comment (srishtigautam21) [reply src/pages/ProductPage/ProductPage.jsx]: Thanks for pointing it out

File: src/component/ProductsFilter/ProductsFilter.jsx [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+      <h3>Category</h3>
+      <div className='filter'>
+        <label htmlFor='checkbox-1'>
+          <input
+            className='margin'
+            id='checkbox-1'
+            name='checkbox'
+            type='checkbox'
+            onClick={() => dispatch({ type: "GRAINS" })}
+          />
```
Comment (Maianki) [inline src/component/ProductsFilter/ProductsFilter.jsx]: Here you can give meaningful values to id and name. For e.g, if the checkbox is for grains you can give id="grains" and name="grains"

File: src/component/ProductsFilter/ProductsFilter.jsx [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+            onClick={() => dispatch({ type: "FRUITS" })}
+          />
+          Fruits
+        </label>
+        <label htmlFor='checkbox-5'>
+          <input
+            className='margin'
+            id='checkbox-5'
+            name='checkbox'
+            type='checkbox'
```
Comment (Maianki) [inline src/component/ProductsFilter/ProductsFilter.jsx]: The same goes for these filters too. Give meaningful name to id and name.

**Review by Maianki — CHANGES_REQUESTED** [review]
Great job. Just had added a few suggestions that can be worked on.

**Review by Maianki — APPROVED** [review]
Looks good you can merge.

**Pull request merged**